codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/EnumGenerator.kt

```diff
@@ -151,8 +155,10 @@ class EnumGenerator(
         meta.render(writer)
         writer.rustBlock("enum $enumName") {
             sortedMembers.forEach { member -> member.render(writer) }
-            docs("$UnknownVariant contains new variants that have been added since this code was generated.")
-            write("$UnknownVariant(String)")
+            if (mode == CodegenMode.Client) {
+                docs("$UnknownVariant contains new variants that have been added since this code was generated.")
+                write("$UnknownVariant(String)")
+            }
```

Comment:
Yes, I understand. They are here because the client generates the same code as the server, plus some more. The one where the server is generating actual different code is in the server only. But not to copy/paste code around, I generate the common code in the client and the additional code in the client. Moving to ServerEnumGenerator would mean to copy the same code there anyway. I am open for suggestions